shinydashboard/lantern/modules/organizationsmodule.R

```diff
@@ -20,13 +21,27 @@ organizationsmodule_UI <- function(id) {
                             Lantern uses the organization name which each endpoint list provides, and the primary and
                             secondary organization names provided by the NPPES NPI data set to match npi organizations to endpoints
                             based on their names and assign a match confidence score. This table shows matches with a match confidence of 97% and up."),
-                        reactable::reactableOutput(ns("npi_orgs_table"))),
+                        reactable::reactableOutput(ns("npi_orgs_table")),
+                        tagList(
+                          h3("Map of Endpoints Linked to an Organization"),
```

Comment:
Now that this information is on the organization page, I think there's some text overlap. I think the first sentence under NPI Organization Matches should read:
```
Endpoints can be linked to organizations in two ways, either by the National Provider Identifier (NPI)
as found in the National Payer and Provider Enumeration System (NPPES), which is preferred,
or by the organization name as reported by a Certified API Developer.
```
And then the first sentence found here should be:
```
This map visualizes the locations of endpoints which Lantern has associated with an organization
name in NPPES.
```
I think that reduces the majority of the repeat information, but feel free to wordsmith as you see fit!